ga4gh/cli/common.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
It might be cleaner to put this stuff into ``cli/__init__.py``. Then you just have `import ga4gh.cli as cli` in every module rather than importing ``cli.common as cli_common``.